crates/bevy_sprite/src/sprite.rs

```diff
@@ -9,10 +9,36 @@ use bevy_render::{
 };
 
 #[repr(C)]
-#[derive(Default, RenderResources, RenderResource)]
+#[derive(RenderResources, RenderResource)]
```

Comment:
As Sprite itself is a RenderResource, I think adding another field to it that has nothing to do with rendering is not a good idea. (The Vertex Shader of a Sprite uses it)

Reply:
Yeah good point, I'll check if it's possible to remake `resize_mode` into ECS component.